addons/calendar/static/tests/tours/calendar_tour.js

```diff
@@ -67,7 +67,7 @@ registry.category("web_tour.tours").add("calendar_appointments_hour_tour", {
         {
             trigger: ".fc-col-header-cell.fc-day.fc-day-mon",
             content: "Check the day is properly displayed",
-            run: "click",
+            run: "hover",
```

Comment:
Note to myself: we might remove all useless `click` someday, as it is default behavior